texar/modules/connectors/connectors.py

```diff
@@ -0,0 +1,866 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Various connectors.
+"""
+
+from typing import (Optional, Union, Callable, Tuple, List,
+                    Any, Dict, TypeVar)
+import numpy as np
+
+import torch
+from torch import nn, split
+
+from texar.hyperparams import HParams
+from texar.modules.connectors.connector_base import ConnectorBase
+from texar.core import layers
+from texar.utils import utils
+from texar.utils import nest
+from texar.utils.types import MaybeTuple
+
+
+# pylint: disable=too-many-locals, arguments-differ
+# pylint: disable=too-many-arguments, invalid-name, no-member
```

Comment:
Same here. `too-many-*` and `invalid-name` checks are already disabled in the global configurations.